docs/en/credential-revocation.rst

```diff
@@ -616,7 +641,244 @@ The Verifier who wants to rely on the mechanism provided by Status Assertion MUS
+
+  Once the Relying Party receives a Digital Credential, this enables it to request the Status List to validate its status through the provided URI parameter and look up the corresponding index. However, the Relying Party is able to store the URI and index of the Digital Credential to request the Status List again at a later time. By doing so regularly, the Relying Party may create a profile of the Digital Credential's validity status. This behaviour might also be abused in cases where this is not intended and unknown to the Holder, e.g. profiling the suspension of a driving license. This behaviour could be mitigated e.g., by regular re-issuance of the Digital Credential.
+
+ Status List Token 
+ .....................
+ 
+ The Credential Issuer MUST make available a Status List Token at the Status List Endpoint. 
+ 
+ .. list-table:: 
+   :widths: 20 60 20
+   :header-rows: 1
+ 
+   * - **Header**
+     - **Description**
+     - **Reference**
+   * - **alg**
+     - A digital signature algorithm identifier such as per IANA "JSON Web Signature and Encryption Algorithms" registry. It MUST be one of the supported algorithms in Section :ref:`Cryptographic Algorithms <supported_algs>` and MUST NOT be set to ``none`` or to a symmetric algorithm (MAC) identifier.
+     - [:rfc:`7515`], [:rfc:`7517`].
+   * -  **typ** 
+     - It MUST be set to ``statuslist+jwt``.
+     - `TOKEN-STATUS-LIST`_
+   * - **kid**
+     - Unique identifier of the Credential Issuer's public key which signs the Status Token.
+     - :rfc:`7638#section_3`.
+ 
+ .. list-table:: 
+   :widths: 20 60 20
+   :header-rows: 1
+ 
+   * - **Payload**
+     - **Description**
+     - **Reference**
+   * - **sub**
+     - REQUIRED. The subject claim MUST specify the URI of the Status List Token. The value MUST be equal to that of the uri claim contained in the status_list claim of the Digital Credential.
+     - [:rfc:`7519`]
+   * - **iat**
+     - REQUIRED. The issued at claim MUST specify the time at which the Status List Token was issued.
+     - [:rfc:`7519`]
+   * - **exp**
+     - REQUIRED. The expiration time claim, if present, MUST specify the time at which the Status List Token is considered expired by the Credential Issuer.
+     - [:rfc:`7519`]
+   * - **ttl**
+     - OPTIONAL. The time to live claim, if present, MUST specify the maximum amount of time, in seconds, that the Status List Token can be cached by a consumer before a fresh copy SHOULD be retrieved. The value of the claim MUST be a positive number encoded in JSON as a number. This amount of time SHOULD NOT exceed the expiration time defined in **exp** claim.
+     - `TOKEN-STATUS-LIST`_
+   * - **status_list**
+     - REQUIRED. JSON Object that contains a Status List.
+     - `TOKEN-STATUS-LIST`_
+ 
```

Comment:
If the compression algorithms are always the same, no problem, but assuming that there could be different compressions, do you think it is a good idea to have a parameter to specify the type of compression used?

Reply:
@grausof this must be motivated and requested in the IETF OAuth Token Status List draft.

Reply:
We agreed to reach draft's editor requesting normative language about the compression alg that must or should be used, currently ZLIB Deflate with a recommended compression level set to 9